### PR TITLE
Use explicit relative imports

### PR DESCRIPTION
**Pull request by Saturn**
Bad idea to use the old relative imports since if you have a duplicate
name somewhere in your module path it might try and import the wrong
module causing an error.

### CODE REVIEW

File: soccer/main.py [hunk 1]
```diff
@@ -3,11 +3,9 @@
...
 import sys
 
-import leagueids
-import teamnames
-
-from soccer.exceptions import IncorrectParametersException
-from writers import get_writer
+from . import leagueids, teamnames
+from .exceptions import IncorrectParametersException
+from .writers import get_writer
```
Comment (ueg1990) [inline soccer/main.py]: why not just use absolute imports since that is recommended in python now

**Comment by Saturn** [comment]
Do you mean like

`from soccer import leagueids, teammates`

or

`from soccer.writers import get_writer`

I see the leading dot style a lot so I just did that. I don't think that style is totally discouraged but there is possibly situations when the absolute import is necessary.

**Comment by ueg1990** [comment]
i see those alot too but apparently its in the PEP8 too where absolute imports are preferred:
http://stackoverflow.com/questions/4209641/absolute-vs-explicit-relative-import-of-python-module

**Comment by Saturn** [comment]
Seems like not everyone likes them but now that the syntax is more standardised a lot of projects do the relative import.

So if it is a matter of style your preference is for absolute imports?

**Comment by ueg1990** [comment]
that is my preference but not a necessity...its more important we follow a consistent format :)

**Comment by Saturn** [comment]
That is true.

@architv can pick a style he prefers.

**Comment by architv** [comment]
@Saturn @ueg1990 I think we should stick with absolute imports since that's the format followed in the package.

**Comment by architv** [comment]
@Saturn Let me know if there is any other issue or we can close the PR.

**Comment by Saturn** [comment]
Changed to use absolute imports. It seems to work for me.

**Pull request merged**